src/types/Job.ts

```diff
@@ -0,0 +1,14 @@
+export interface Job {
```

Comment:
Create separate types for each of them:

```suggestion
export interface Enumeration {
  id: String;
  name: String;
  description: String;
  typeId: String;
}
export interface TemporalExpression {
  id: String;
  description: String;
}
export interface Status {
  id: String;
  description: String;
}
export interface Job {
  id: String;
  name: String;
  systemJobEnum: Enumeration;
  parentJobId: String;
  runTime: String;
  serviceName: String;
  status: Status;
  tempExpr: TemporalExpression;
  currentRetryCount: number;
}
```

Reply:
@shashwatbangar